src/Tonic/Noise.cpp

```diff
@@ -14,9 +14,9 @@
 namespace Tonic {
   namespace Tonic_ {
   
-    PinkNoise_::PinkNoise_() : pinkCount_(1), pinkAccum_(0) {
+    PinkNoise_::PinkNoise_() :pinkAccum_(0),  pinkCount_(1) {
```

Comment:
@andik Would you mind explaining the purpose of this change and the other similar changes, where the initilizer lists are reordered?